### PR TITLE
Code page cleanup for front-end libraries

### PR DESCRIPTION
**Pull request by robertjd**
- Add links to sample applications
- Clarify the difference between quickstarts, samples, and guides
- Add code page for Vue

### CODE REVIEW

File: _source/_code/react/index.md [hunk 1]
```diff
@@ -7,19 +7,27 @@ integration: client
...
   </li>
+  <li>
+    <a href='https://github.com/okta/samples-js-react' class='code-button'>
+      <span class='fa fa-github'></span><span>React Sample Applications</span>
+    </a>
+  </li>
 </ul>
 
-## More Resources
+## Guides
```
Comment (jmelberg-okta) [inline _source/_code/react/index.md]: If we're changing these headings, lets make sure we update the .NET, Java, PHP, and Python files too.

File: _source/_code/vue/index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (jmelberg-okta) [inline _source/_code/vue/index.md]: nit: We should probably **Vue** to our `/documentation` icon page, as the only way to get to this page is via the TOC.

I think it is out of the scope of this PR - but we should talk about when we should start building out the redesigns from @alexdahl-okta.

Comment (alexdahl-okta) [reply _source/_code/vue/index.md]: I have an off-the-radar plan to clean up some of the front `/documentation` page shortly... happy to add Vue if it doesn't make this PR.

**Comment by robertjd** [comment]
I've pushed a new commit that touches all the code pages, bringing them closer to the new design the @alexdahl-okta is working on.

File: _source/_code/angular/index.md [hunk 3]
```diff
@@ -7,19 +7,24 @@ integration: client
 
 # Add Okta authentication to your Angular app
 
-New to Okta? Our Quick Start Guide will walk you through adding user authentication to your Angular application in minutes.
+New to Okta? Our Quick Start Guide will walk you through adding user authentication to your Angular application in minutes,
+or download a complete sample application to see our recommended usage.
```
Comment (jmelberg-okta) [inline _source/_code/angular/index.md]: nit: In general, most of our documentation doesn't include line breaking for readability on editors.

**Pull request merged**